source/FpsConfig.h

```diff
@@ -71,7 +71,12 @@ class PlayerConfig {
 	Vector2			turnScale = Vector2(1.0f, 1.0f);			///< Player (horizontal/vertical) turn rate scaler
 	Array<bool>		axisLock = { false, false, false };			///< World-space player motion axis lock
 	bool			stillBetweenTrials = false;					///< Disable player motion between trials?
-	bool			resetPositionPerTrial = false;				///< Reset the player's position on a per trial basis (to scene default)
+	bool			resetPositionPerTrial = false;				///< Reset the player's position on a per trial basis (to scene default)'
+	float			sprintMultiplier = 2.0f;					///< Multiplies with players move rate when player is sprinting (defaults to 2x)
+	float			headBobAmplitude = 0.1f;					///< Determines how high/low players head will oscillate during movement (defaults to 0.1)
+	float			headBobFrequency = 1.0f;					///< Determines how fast players head will oscillate during movement (defaults to 1)
```

Comment:
If there's a toggle to turn off headbob that indicates that head bob is off by default, I'm okay with sensible defaults for the bob parameters. It's much easier for people to start with something sensible when they turn on head bob than to have to manually adjust the parameters every time they try to turn it on.

Of course if we want it always enabled and to use specific values here to "disable" it, then the values should be disabled, but then we'll want an example in the docs of good default values and maybe even a sample in `data-files/samples`.

Reply:
I'm fine with the enabled flag, though the historical model in FPSci is to set parameter values to 0 to disable effects instead of creating an additional boolean toggle (unless required). For example setting `moveRate = 0` to disable motion instead of having an additional `playerMoveEnabled` flag.

Reply:
There's a mix of `enable` and set-to-0 parameters in FPSci. I wouldn't say that one happens more commonly than others and it's more of a "what makes the most sense" decision.

I'd almost argue that `moveRate` is a special case since movement is an "always on" function while head bob seems more like an optional thing that I'd prefer to fully disable (execute no code) when it's off. Though the code may be inconsequential in runtime at the moment, so it probably doesn't matter.

Another example is the HUD where we don't just set the alpha to 0 to disable it, but we have a toggle to disable everything at once, then a bunch of parameters to customize it when on.

Reply:
I have created flags for both Acceleration and Headbob and kept them both disabled by default. Let me know if it needs any changes.

Reply:
Only (super minor) remaining fix here is you added a ''' at the end line 74 you can remove

Reply:
Updated.

Reply:
It appears the `'` is still at line 74

Reply:
Sorry may have pressed undo. Fixed.